docs/data-sources.md

```diff
@@ -8,6 +8,13 @@ Mapzen Terrain Tiles are powered by several major open data sets and we owe a tr
 
 The underlying data sources are a mix of:
 
+- [CDEM](http://geogratis.gc.ca/api/en/nrcan-rncan/ess-sst/c40acfba-c722-4be1-862e-146b80be738e.html) (Canadian Digital Elevation Model) in Canada, with variable spatial resolution (from 20-400 meters) depending on the latitude.
```

Comment:
Sounds good! It seemed to be ordered by resolution, so I was going with that. Alpha it is!